### PR TITLE
Added currency formats

### PR DESCRIPTION
**Pull request by bigkraig**
Added currency formats so that you can plot graphs in the most important format, money.

### CODE REVIEW

File: public/app/components/kbn.js [hunk 1]
```diff
@@ -407,6 +409,26 @@ function($, _, moment) {
...
+    if (size === null) { return ""; }
+
+    if (Math.abs(size) < Math.pow(10, 3)) {
+      return currency + Intl.NumberFormat().format(kbn.toFixed(size, 2));
+    }
+    if (Math.abs(size) < Math.pow(10, 6)) {
+      return currency + Intl.NumberFormat().format(kbn.toFixed(size/Math.pow(10, 3), 3)) + "K";
+    }
+    else if (Math.abs(size) < Math.pow(10, 9)) {
+      return currency + Intl.NumberFormat().format(kbn.toFixed(size/Math.pow(10, 6), 3)) + "MM";
```
Comment (torkelo) [inline public/app/components/kbn.js]: is MM a standard abbreviation for million when dealing with currency?

Comment (bigkraig) [inline public/app/components/kbn.js]: It's pretty common

Comment (bigkraig) [inline public/app/components/kbn.js]: @torkelo I'm willing to changing it to M or whatever to get this merged in

**Comment by rex** [comment]
It's really unfortunate that this PR stagnated. Having these formatting types would be tremendously helpful for visual presentation. 

It's especially painful because the reason the AppVeyor check failed appears to be unrelated to the code itself. It appears to have failed because of a hash mismatch on the LESS npm package. 

```
lient\node_modules\ws\build\Release\validation.exp
  Generating code
  Finished generating code
  validation.vcxproj -> C:\gopath\src\github.com\grafana\grafana\node_modules\karma\node_modules\socket.io\node_modules\socket.io-client\node_modules\ws\build\Release\\validation.node
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4
 
npm ERR! shasum check failed for C:\Users\appveyor\AppData\Local\Temp\npm-1220-ea071131\registry.npmjs.org\less\-\less-1.4.2.tgz
npm ERR! Expected: b7deefe98a3a87bee364411b3df2d1efe5a412d0
npm ERR! Actual:   254366a3b976b0cdca2a4bbb337a897f0c287785
npm ERR! From:     https://registry.npmjs.org/less/-/less-1.4.2.tgz
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
 
npm ERR! Please include the following file with any support request:
npm ERR!     C:\gopath\src\github.com\grafana\grafana\npm-debug.log
Command exited with code 1
```

@bigkraig @torkelo Is there any chance someone could attempt the build again so that it can pass and this could get merged in?

**Comment by torkelo** [comment]
its looking ok, please rebase the PR against master and add unit tests for the new formats

**Comment by bigkraig** [comment]
I'm pretty green with javascript. I've written some unit tests but I'm unable to run them due to what looks like a simple "include" type of issue:

Running "karma:test" (karma) task
INFO [karma]: Karma v0.12.37 server started at http://localhost:9876/
INFO [launcher]: Starting browser PhantomJS
INFO [PhantomJS 1.9.8 (Mac OS X 0.0.0)]: Connected on socket Ov1DVBxOYhZsXKkA4baa with id 55046568
............
PhantomJS 1.9.8 (Mac OS X 0.0.0) value format: pounds should translate 0.99 as £0.99 FAILED
	ReferenceError: Can't find variable: Intl

**Comment by rex** [comment]
Well, I guess that's that. Absolute shame. @torkelo is there anything preventing this from a merge?

**Comment by bigkraig** [comment]
@rex it's in there already, a different set of commits were merged in.

**Comment by rex** [comment]
Oh awesome @bigkraig! That's great news! I was sad about it because it is so damn useful and I'm glad now to hear that it's in there :)

**Pull request closed without merging**